.ci/linux/build.sh

```diff
@@ -4,7 +4,4 @@
 #
 set -euxo pipefail
 
-# Remove Full Framework projects
-.ci/linux/remove-projects.sh
-
-dotnet build
+./build.sh
```

Comment:
nit: I think we can also remove the comment from line 3.

Reply:
done!